indexer/src/main/java/org/pdxfinder/commands/dataloaders/LoadIRCC.java

```diff
@@ -75,108 +58,71 @@ public void run(String... args) throws Exception {
         parser.accepts("loadALL", "Load all, including IRCC PDX data");
         OptionSet options = parser.parse(args);
 
-
         if (options.has("loadIRCC") || options.has("loadALL")) {
-
             initMethod();
-
             irccAlgorithm();
-
         }
     }
 
 
     public void irccAlgorithm() throws Exception {
 
         step00StartReportManager();
-
         step02GetMetaDataJSON();
-
         step03CreateProviderGroup();
-
         step04CreateNSGammaHostStrain();
-
-        step05CreateNSHostStrain();
-
         step06CreateProjectGroup();
-
         step07GetPDXModels();
 
-
         for (int i = 0; i < jsonArray.length(); i++) {
-
             this.jsonData = jsonArray.getJSONObject(i);
 
             if (loadedModelHashes.contains(jsonData.toString().hashCode())) continue;
             loadedModelHashes.add(jsonData.toString().hashCode());
 
             step08GetMetaData();
-
             step09LoadPatientData();
-
             step10LoadExternalURLs();
-
-            step11LoadBreastMarkers();
-
             step12CreateModels();
-
             step13LoadSpecimens();
-
-            step14LoadPatientTreatments();
-
             step17LoadModelDosingStudies();
-
             step16LoadVariationData();
-
+            addAccessModality();
```

Comment:
Is this another step in the generic process?  If so, might want to rename to step17AddAccessModality (or something)

Reply:
Done - renamed to `step18SetAccessGroup()`